includes/API.php

```diff
@@ -129,12 +129,20 @@ public function find_product_item( $catalog_id, $retailer_id ) {
 	 *
 	 * @since 2.0.0-dev.1
 	 *
-	 * @param string $product_group_id
-	 * @param array $data
+	 * @param string $product_group_id parent product ID
+	 * @param array $data product data
+	 * @return Response
+	 * @throws Framework\SV_WC_API_Exception
 	 */
 	public function create_product_item( $product_group_id, $data ) {
 
-		// TODO: Implement create_product_item() method.
+		$request = $this->get_new_request( [ $product_group_id, '/products', 'POST' ] );
```

Comment:
I'm not a fan of how `get_new_request()` is set up with the arguments array. At the very least it should be a keyed array so there's some clarity IMO. I know it's not something about this PR though :) I left a comment on another story, hopefully we can fix it.